hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/lock/OmReadOnlyLock.java

```diff
@@ -17,48 +17,25 @@
 
 package org.apache.hadoop.ozone.om.lock;
 
+import org.apache.hadoop.ozone.om.lock.OzoneManagerLock.Resource;
+
 /**
  * Read only "lock" for snapshots
  * Uses no lock.  Always returns true when aquiring
  * read lock and false for write locks
  */
 public class OmReadOnlyLock implements IOzoneManagerLock {
```

Comment:
not related to this PR but this class should be renamed to something akin to `OmDummyReadLock`

Reply:
Will do it in the next PR, this one is aged.